connector/gradle/libs.versions.toml

```diff
@@ -10,13 +10,20 @@ testcontainers = "1.20.0"
 [libraries]
 # upstream EDC dependencies
 edc-vault-hashicorp = { module = "org.eclipse.edc:vault-hashicorp", version.ref = "edc" }
+edc-web-spi = { module = "org.eclipse.edc:web-spi", version.ref = "edc" }
+edc-core-spi = { module = "org.eclipse.edc:core-spi", version.ref = "edc" }
+edc-auth-spi = { module = "org.eclipse.edc:auth-spi", version.ref = "edc" }
+edc-data-plane-spi = { module = "org.eclipse.edc:data-plane-spi", version.ref = "edc" }
+edc-data-plane-public-api = { module = "org.eclipse.edc:data-plane-public-api-v2", version.ref = "edc" }
```

Comment:
please note that this module will be deprecated starting from version 0.12.0, so in the future there will be the need for a custom proxy implementation. ([ref.](https://github.com/eclipse-edc/Connector/tree/1a700a7e4cc1541fea1ad9dc6193b80862ed12b4/docs/developer/decision-records/2025-02-07-http-proxy-data-plane-deprecation))